swap/swap.go

```diff
@@ -371,6 +396,47 @@ func (s *Swap) handleEmitChequeMsg(ctx context.Context, p *Peer, msg *EmitCheque
 	return err
 }
 
+func (s *Swap) handleConfirmChequeMsg(ctx context.Context, p *Peer, msg *ConfirmChequeMsg) {
+	p.lock.Lock()
+	defer p.lock.Unlock()
+	cheque := msg.Cheque
+
+	if p.getPendingCheque() == nil {
+		p.logger.Warn("ignoring confirm msg, no pending cheque", "got", cheque)
```

Comment:
what does the `got` key represent here? not sure about the wording in this case.

we can maybe be more explicit, something like:
> `p.logger.Warn("ignoring confirm msg, no pending cheque", "confirm message cheque", cheque)`

Reply:
done

Reply:
Same as further above.

Reply:
> see [here](https://github.com/ethersphere/swarm/pull/1887#discussion_r340618967), although for this case i tend to agree on logging what was expected versus what was received

In this particular case nothing was expected.

Reply:
since the `String` function for `Cheque` does not include the workd `cheque` (which is good, i think) i'm wondering if a message like:

> `INFO [11-04|16:24:04.365] ignoring confirm msg, no pending cheque | got=Contract: 7d707c773c0a69ca7c3e2c3f8839ca363b6cb50c Beneficiary: c45d64d8f9642a604db93c59fd38492b262391ca CumulativePayout: 2250895983922952680 Honey: 1500576353024385140`

is clear without saying that `got` actually refers to a cheque (for the case that there was nothing expected).

what do you think?

Reply:
That's why i changed `got` to `confirm message cheque`. Did I miss it somewhere?

Reply:
no, _i_ was the one who missed it. my bad!